mobile/src/main/java/org/openhab/habdroid/util/ExtensionFuncs.kt

```diff
@@ -309,6 +309,14 @@ fun Context.getHumanReadableErrorMessage(url: String, httpCode: Int, error: Thro
     }
 }
 
+fun Context.openInAppStore(app: String) {
```

Comment:
`openAppInPlayStore()`?

Reply:
Any app store installed on the device can be opened, not only the play store. E.g. G-Droid (an alternative F-Droid client) supports rating, so it makes sense to offer opening in other app stores as well.

Reply:
So those other apps also register for the `market` scheme? If so, LGTM then.

Reply:
Yes, they do. On my phone I can choose between F-Droid and Play Store.